### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by eyalk11** [ahmedkhalf/project.nvim#133]
Hi,
The current documentation is lacking. 

`AddProject` is not even mentioned. 
I didn't understand why upon `Telescope projects` and `enter` it request me to find files. 
What files are assigned to what project? is it open buffers? is the lsp workspace variable kept?

**Comment by andyl** [comment]
Bingo

**Comment by yrahul3910** [comment]
Yes, this took me quite a while to figure out. Contrary to what the README suggests, simply calling `.setup()` doesn't seem to suffice; you also need to call `.init()` afterwards:

```lua
require("project_nvim").setup()  -- you can add options here, I use defaults for now
require("project_nvim.project").init()
```

With this, Telescope picks up my projects for me in the search.

**Comment by DrKJeff16** [comment]
@andyl @yrahul3910 @eyalk11

I've addressed these issues in my new, improved and maintained `project.nvim`, as stated by the commit above referencing this issue.
https://github.com/DrKJeff16/project.nvim

See the [`:ProjectAdd` Section](https://github.com/DrKJeff16/project.nvim#projectadd) in the `README`.

I've also fixed the `init()` issue by default.
